administration/console.md

```diff
@@ -16,23 +16,29 @@ The console offers the option to:
 ## Accessing the Console
 
 Accessing the console depends on the start mode of openHAB.
+
+### Started with Script
 The manually start using shell/batch script ends directly in the console.
```

Comment:
You didn't add this sentence but could you still improve it? As an outsider, I would not understand the meaning...